metals/src/main/scala/scala/meta/internal/metals/FindTextInDependencyJarsRequest.scala

```diff
@@ -0,0 +1,6 @@
+package scala.meta.internal.metals
+
+case class FindTextInDependencyJarsRequest(
+    mask: String,
```

Comment:
Should we instead have `include` and `exclude` the same as normal search so that it's compatible with new proposed text search API? We can use include for now for the mask.

Also instead of content we could model it on the VS Code API from https://github.com/Microsoft/vscode/blob/f35b6ce8d8a24f2dcf756d8bd6e2069dd4f25cc9/src/vs/vscode.proposed.d.ts#L21

And essentially change everything to
```scala

case class FindTextInDependencyJarsRequest(
    @Nullable
    options: FindTextInFilesOptions,
    query: TextSearchQuery
)

case class TextSearchQuery(
   pattern: String,
   @Nullable
   isRegExp: java.lang.Boolean,
   @Nullable
   isCaseSensitive:  java.lang.Boolean,
   @Nullable
   isWordMatch:  java.lang.Boolean;
)

case class FindTextInFilesOptions(
 @Nullable
  include: String,
  @Nullable
  exclude: String
)
```

Almost everything here is nullable, so we could implement it later, but it would be good to have an API we can use later on.